Expensify/App
# [WIP] Setup repo for composite run steps actions.

### Details
Just a POC for composite run steps actions in this repo. A first-step towards a greater suite of improvements in our GH actions and workflows. Created a directory structure like this:

```
.github/
    actions/
        javascript/
            ... all our existing actions
        composite/
            ... new composite action. Must be a file called `action.yml` inside a directory named for the action.
    workflows/
    ...
```

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/174189

### Tests
1. Verify that PR checks pass on this PR.
1. Merge this PR.
1. Verify that other GH actions keep working. We'll notice if they don't 🙃 
1. Purposely run a workflow with an action that we know will fail. For example, attempt to manually CP a PR w/ a number that belongs to an issue instead of a pull request. That will definitely fail and should ping in #announce.

### QA Steps
None.
